ts/components/HorizontalScroll.tsx

```diff
@@ -46,7 +49,24 @@ const styles = StyleSheet.create({
 export const HorizontalScroll: React.FunctionComponent<Props> = (
   props: Props
 ) => {
-  const animVal = new Animated.Value(0);
+  const animVal = new Animated.Value(
+    props.indexToScroll
+      ? props.indexToScroll * Dimensions.get("window").width
+      : 0
+  );
+  const scrollRef = React.useRef<ScrollView>(null);
+
+  React.useEffect(() => {
```

Comment:
If we props.indexToScroll is undefined we could avoid to execute setTimeout without code execution
Could it be a refactoring?
const scrollOffset =
    (props.indexToScroll ?? 0) * Dimensions.get("window").width;
  const animVal = new Animated.Value(scrollOffset);
  const scrollRef = React.useRef<ScrollView>(null);

```typescript
  React.useEffect(() => {
    fromNullable(props.indexToScroll).map(_ => {
      setTimeout(() => {
        if (scrollRef.current) {
          scrollRef.current.scrollTo({
            x: scrollOffset,
            y: 0,
            animated: false
          });
        }
      }, 0);
    });
  }, [scrollRef]);
```

Reply:
yup! It surely sounds good i'll add it to next commit